format/mbr/mbr.go

```diff
@@ -0,0 +1,72 @@
+package mbr
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/registry"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+//go:embed mbr.jq
+var mbrFS embed.FS
+
+func init() {
+	registry.MustRegister(decode.Format{
```

Comment:
This was refactored a bit in master, is now `interp.RegisterFormat(...)`

Reply:
Will rebase and fix